src/category_theory/limits/shapes/images.lean

```diff
@@ -696,6 +696,7 @@ instance strong_epi_mono_factorisation_inhabited {X Y : C} (f : X ⟶ Y) [strong
 
 /-- A mono factorisation coming from a strong epi-mono factorisation always has the universal
     property of the image. -/
+@[simps]
```

Comment:
I would prefer if we didn't have this `simps` tag. I believe that any result that relies on how the specific image factorisations are constructed here should have a more conceptual proof that just relies on the properties `lift_fac` and `fac_lift`. For example, in my opinion, the "correct" proof for the proof of `is_iso (coimage_image_comparison f)` in `abelian/basic.lean:366` is something like (module Lean style topics)
```lean
instance : is_iso (coimage_image_comparison f) :=
begin
  convert is_iso.of_iso (is_image.iso_ext (coimage_strong_epi_mono_factorisation f).to_mono_is_image
    (image_strong_epi_mono_factorisation f).to_mono_is_image),
  ext,
  change _ = _ ≫ (image_strong_epi_mono_factorisation f).m,
  simp [-image_strong_epi_mono_factorisation_to_mono_factorisation_m]
end
```
This just invokes the universal property rather than going through the construction of images from strong epi-mono factorisations.

Reply:
Thanks, I'm convinced!